packages/synapse-react-client/test/lib/containers/entity_finder/details/DetailsView.integration.test.tsx

```diff
@@ -308,7 +308,7 @@ describe('DetailsView tests', () => {
         setSort: mockSetSort,
       })
 
-      await userEvent.click(screen.getAllByRole('button')[2])
+      await userEvent.click(screen.getAllByRole('button')[1])
```

Comment:
Adjust index for ascending / descending buttons  -- in MarkdownPopover, converted the span with `role="button"` to a MUI Box, so there is one fewer "button" in this component than there was previously.

Reply:
Reverted this change and added `role="button"` to the MUI Box to maintain the expectation that the MarkdownPopover provides a button role and its children don't have to include a button (as in HelpPopover and in CertificationQuiz)